src/lib.rs

```diff
@@ -275,13 +316,14 @@ impl VadSession {
     /// # Panics
     ///
     /// If the range is out of bounds of the speech buffer this method will panic.
-    pub fn get_speech(&self, start: usize, end: Option<usize>) -> &[f32] {
-        let speech_start = start * (self.config.sample_rate / 1000);
-        if let Some(speech_end) = end {
-            let speech_end = speech_end * (self.config.sample_rate / 1000);
-            &self.session_audio[speech_start..speech_end]
+    pub fn get_speech(&self, start_ms: usize, end_ms: Option<usize>) -> &[f32] {
```

Comment:
So I think you want an assert here instead of just relying on a panic, because underflow will happen in release model and then you'll be doing a negative range which I guess should still panic but it'll be more confusing a panic